measure/src/test/scala/org/apache/griffin/measure/utils/HttpUtilTest.scala

```diff
@@ -0,0 +1,63 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+package org.apache.griffin.measure.utils
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class HttpUtilTest extends FlatSpec with Matchers {
+  val url = "http://10.148.181.248:39200/griffin/accuracy"
+  val method = "post"
+  val user = ""
+  val password = ""
+  val params = Map[String, Object]()
+  val header = Map[String, Object](("Content-Type", "application/json"))
+  val data = "{}"
+
+  "postData httpReq" should "be with auth" in {
+    try {
+      HttpUtil.postData(url, user, password, params, header, data)
+    } catch {
+      case e: Throwable => println("cannot connect http client." + e)
+    }
+  }
```

Comment:
All the test cases are not verified, which is meaningless in unit test.

Reply:
but I find if I mock one local variable, I need to use powermock, and it will change pom dependency.

Reply:
I don't think there's any static or private function you need to mock, powermock is unnecessary.
`httpReq.postData(data).asString` is not easy to test here, you only need to mock for this line.
BTW, the verification is expected for both success response and parameter existence.

Reply:
in scala, how to mock httpReq.postData(data).asString, could you give an example

Reply:
or something like when... then return..., not familar with scala test.

Reply:
the pattern is not your limit, we're talking about the goal of UT, just to prove your code could accept "username" and "password" and works well.

Reply:
ok, got it.